.eslintrc

```diff
@@ -1,5 +1,13 @@
 {
     "extends": "./node_modules/@connectedcars/setup/.eslintrc",
     // Force eslint to use local installed parser not the one from ./node_modules/@connectedcars/setup
-    "parser": "@typescript-eslint/parser"
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["jest"],
+    "rules": {
+      "jest/no-disabled-tests": "warn",
+      "jest/no-focused-tests": "error",
+      "jest/no-identical-title": "error",
+      "jest/prefer-to-have-length": "warn",
+      "jest/valid-expect": "error"
+    }
```

Comment:
If we want these linting rules, shouldn't they be added in node-setup instead? Perhaps create a separate story to move these rules to node-setup.

Reply:
I didn't move them into node-setup because I wasn't sure if we wanted them or not (I personally think they are helpful) but I'll add them (https://app.shortcut.com/connectedcars/story/114847/add-eslint-plugin-jest-to-node-setup).